tests/test_theme.py

```diff
@@ -380,6 +382,14 @@ def test_repr(default_theme):
     assert str(default_theme.colorbar_orientation) in rep
     assert default_theme._name.capitalize() in rep
 
+    # verify that the key for each line in the repr is less than the minimum
+    # key size. This makes sure that any new keys are either less than the size
+    # of the key in the repr or the key length is increased
+    for line in rep.splitlines():
+        if ':' in line:
+            pref, *rest = line.split(':', 1)
+            assert pref.endswith(' '), f"Key str too long or need to raise key length:\n{pref!r}"
```

Comment:
I didn't notice the auto-merge (I didn't expect it right after a set of changes), so I pushed a change here to print the offending key too. Had to refactor a bit in order to get the assert on one line. Might want to disable auto-merge until this is reviewed (I can't disable it for some reason; I could do that earlier on other PRs).

Reply:
Agree with change. Thanks for fixing that.

Reply:
OK, thanks, this is good to merge then. Assuming doc builds now (it was red which blocked auto-merge earlier).